itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosExternalTables.java

```diff
@@ -436,16 +438,29 @@ public void externalTableIncrementalReplication() throws Throwable {
     }
 
     List<String> loadWithClause = externalTableBasePathWithClause();
-    replica.load(replicatedDbName, primaryDbName, loadWithClause)
+    replica.load(replicatedDbName, primaryDbName, withClause)
         .run("use " + replicatedDbName)
         .run("show tables like 't1'")
         .verifyResult("t1")
         .run("show partitions t1")
         .verifyResults(new String[] { "country=india", "country=us" })
         .run("select place from t1 order by place")
-        .verifyResults(new String[] { "bangalore", "mumbai", "pune" })
+        .verifyResults(new String[] {})
```

Comment:
Why is there no data here?

Reply:
Since the modification happened after dump, data will be seen only after next dump-load cycle, which is being tested.

Reply:
The copy was happening in repl load, change was prior to that.

Reply:
Because copy is happening during load.